src/players/services/players.service.ts

```diff
@@ -113,6 +113,16 @@ export class PlayersService implements OnModuleInit {
   }
 
   async createPlayer(steamProfile: SteamProfile): Promise<Player> {
+    const superUserId = this.environment.superUser;
+
+    if (steamProfile.id === superUserId) {
+      return await this.forceCreatePlayer({
+        name: steamProfile.displayName,
+        steamId: steamProfile.id,
+        roles: [PlayerRole.superUser, PlayerRole.admin],
+      } as any);
```

Comment:
Why is `as any` needed here?

Reply:
Roles isn't apart of the interface, but the method accepts a rest param when creating. It's to get around type, otherwise, we could expand the type to allow any field from user model

Reply:
Consider expanding the param type, then :)